.gitlab/deploy_containers/deploy_containers_a7.yml

```diff
@@ -95,8 +101,24 @@ deploy_containers-a7_internal-rc:
     VERSION: 7-rc
 
 
-deploy_containers-ot:
+deploy_containers-a7-ot_internal:
   extends: .deploy_containers-a7-base-ot
+  rules:
+    !reference [.on_internal_final-ot]
+  before_script:
+    - if [[ "$VERSION" == "" ]]; then VERSION="$(inv agent.version --major-version 7 --url-safe --pipeline-id $PARENT_PIPELINE_ID)" || exit $?; fi
```

Comment:
nitpicky, but could you drop the `--agent-version 7`? They already default to 7 and we plan on removing those parameters soon